doc/Language/5to6-perlvar.pod6

```diff
@@ -48,21 +44,17 @@ you can call methods on it. Unlike C<$_> you cannot assume C<@_> as the
 default variable for those methods to operate on (i. e. C<@_.shift> works,
 C<.shift> does not).
 
-=head3 $LIST_SEPARATOR, $"
+=head3 X<$LIST_SEPARATOR, $"|Foreign,$" (Perl)>
```

Comment:
In all these, you're dropping the first term in the list, or the alias. As a matter of fact, these were not indexed before, but if you index them, you should use both terms.

Reply:
Yes, those pages are a bit troublesome in general. I had to go through hundreds and hundreds of anchors when doing this, so some things certainly slipped, and I also decided to work on them when the main pack is done.